src/main/java/io/stargate/sgv2/jsonapi/config/constants/RerankingConstants.java

```diff
@@ -2,15 +2,10 @@
 
 public interface RerankingConstants {
 
-  interface RerankingColumn {
+  interface CollectionRerankingOptions {
     String ENABLED = "enabled";
-    String SERVICE = "service";
+    String SERVICE = CommonConstants.ServiceConfig.SERVICE;
```

Comment:
Hmmmh. Constant pointing to a constant, to be used as... constant of some kind?
I am not sure these make tons of sense.